src/server/Elsa.Server.Api/Endpoints/Workflows/Test.cs

```diff
@@ -0,0 +1,57 @@
+using Elsa.Server.Api.ActionFilters;
+using Elsa.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Threading;
+using System.Threading.Tasks;
+using Elsa.Models;
+
+namespace Elsa.Server.Api.Endpoints.Workflows
+{
+    [ApiController]
+    [ApiVersion("1")]
+    [Route("v{apiVersion:apiVersion}/workflows/{workflowDefinitionId}/{version}/{signalRConnectionId}/test")]
+    [Produces("application/json")]
+    public class Test : Controller
```

Comment:
This class looks very similar to `Execute`, the main difference being the presence of the SignalR connection ID. Can you merge these two classes into one, perhaps making the SignalR connection ID optional to indicate that it needs to be setup?

Reply:
That one was original endpoint and later moved to it's separate WorkflowTest/Execute endpoint. Deleted.